src/java/fr/paris/lutece/portal/web/upload/UploadServlet.java

```diff
@@ -112,7 +112,7 @@ protected void doPost( HttpServletRequest req, HttpServletResponse response ) th
         String strResultJson = objectMapper.writeValueAsString( mapJson );
         if ( AppLogService.isDebugEnabled( ) )
```

Comment:
It is unnecessary to check if logging is enabled because it is already managed with the debug level.